client/src/components/LevelLayout.tsx

```diff
@@ -25,6 +27,19 @@ const LevelLayout: React.FC<Props> = ({
   children,
 }) => {
   const game = useGame();
+  const toast = createStandaloneToast();
```

Comment:
Is there a reason why you opted for `createStandaloneToast()` instead of `useToast()`? 

From the [docs](https://chakra-ui.com/docs/feedback/toast#standalone-toasts):
> If you're using a custom theme, you must pass it in the arguments to createStandaloneToast. If you don't, the default theme will be applied, causing the font size of your page to shift when the toast is open.

This has no difference now as we are not using any customized theme. If there is no specific reason for using a stand alone toast, I think we should use `useToast` so that, if in the future we add a custom theme, the toast will reflect those changes.

Reply:
You're right, I don't think there is any reason not to use `useToast()`